app/services/transcript_service.py

```
@@ -93,34 +93,83 @@ def __init__(self, use_cache: bool = True):
                 self.logger.info(f"Using cached transcript for video {video_id}")
                 return cached_transcript
 
+        transcript = None
+        
         try:
-            transcript_list = YouTubeTranscriptApi.get_transcript(video_id)
+            # Try to get transcript with specified languages
+            transcript_list = YouTubeTranscriptApi.get_transcript(video_id, languages=languages)
             transcript = self._format_transcript(transcript_list)
+            self.logger.info(f"✅ Got transcript in {languages} for video {video_id}")
         except Exception as e:
```

Ruff (0.11.9)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)